packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -82,8 +82,8 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
     bool internal subgameAtRootResolved;
 
     /// @notice Semantic version.
-    /// @custom:semver 0.0.11
-    string public constant version = "0.0.11";
+    /// @custom:semver 0.0.12
+    string public constant version = "0.0.12";
```

Comment:
Very nice catch.

Reply:
These are the kinds of things a real seasoned contract developer gets right... once the tests fail and they go searching through a bunch of docs to work out what's going on, then have about 5 gos at it. 😆